internal/pkg/cli/env_init.go

```diff
@@ -50,9 +52,21 @@ var (
 
 type initEnvVars struct {
 	*GlobalOpts
-	EnvName      string // Name of the environment.
-	EnvProfile   string // AWS profile used to create an environment.
-	IsProduction bool   // Marks the environment as "production" to create it with additional guardrails.
+	Name         string
+	Profile      string
+	IsProduction bool
+	UseDefault   bool
+
+	ImportVPC struct {
+		ID               string
+		PublicSubnetIDs  []string
+		PrivateSubnetIDs []string
+	}
+	AdjustVPC struct {
+		CIDR               net.IPNet
```

Comment:
Is there a benefit to using this type over a `string`, for example will we give the user immediately an error if they enter an invalid CIDR range with this type?

Reply:
yes for example:

`$ copilot env init --override-vpc-cidr 10.10.10.10`

```
✘ invalid argument "10.10.10.10" for "--override-vpc-cidr" flag: invalid CIDR address: 10.10.10.10
```